docs/data-misc-tables.rst

```diff
@@ -0,0 +1,84 @@
+Misc Tables
+===========
+
+.. Enforce HTML output for pd.Series
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+
+    import audformat
+
+
+    audformat.core.common.format_series_as_html()
+
+
+A misc table or miscellaneous table links labels to an index.
+The index has no restrictions,
+but should not contain duplicated entries.
+The arbitrary number of index columns of a misc table
```

Comment:
```suggestion
An arbitrary number of index columns of a misc table
```

Reply:
If we change to "An arbitrary" I'm afraid that the meaning changes. Now it could also mean that after some index columns we store some label columns. But we need to emphasize, that we first store all index columns and then the label columns.

So, maybe rephrasing to:

```rst
The index has no restrictions
and can contain an arbitrary number of columns (called levels),
but should not contain duplicated entries.
In the CSV file of the misc table,
the index columns are stored in front of the label columns.
```

Reply:
Yes, your suggestion sounds good.

Reply:
Changed accordingly in https://github.com/audeering/audformat/pull/195/commits/6e8000f3f24a376943bed7b7fd003b555447c528